src/scss/_buttons.scss

```diff
@@ -19,6 +19,13 @@
   }
 }
 
+.btn.disabled,
```

Comment:
Looks fine but you might want to put this after the `@each $sm, $value in $sm-colors {` loop below as the disabled style isn't applied to the social media colour buttons.

Reply:
I placed it at the end so we are sure to keep it relevant as it should have priority

Reply:
thanks